content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -60,7 +60,7 @@ spec:
   # Configure a topology spread constraint
   topologySpreadConstraints:
     - maxSkew: <integer>
-      minDomains: <integer> # optional; beta since v1.25
+      minDomains: <integer> # optional; GA since v1.30
```

Comment:
Because this has been beta for long enough and all Kubernetes supported releases will recognize that field:
```suggestion
      minDomains: <integer> # optional
```

could also have:
```suggestion
      minDomains: <integer> # optional; generally available since v1.30
```